project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
I did this in https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/pull/132 and released a new tk-jetty9 from it, 1.5.2.  I'll close this PR out and open a new one that bumps the tk-jetty9 dependency to 1.5.2.